src/09_dictionaries.py

```diff
@@ -36,12 +36,28 @@
 # Add a new waypoint to the list
 # YOUR CODE HERE
 
+waypoints.append(
+    {
+    "lat": 42,
+    "lon": -124,
+    "name": "the best place"    
+    }
+)
+print(waypoints)
+
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
 
 # YOUR CODE HERE
+waypoints[0]["name"] = "not a real place"
+waypoints[0]["lon"] = -130
 
 # Write a loop that prints out all the field values for all the waypoints
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+for place in waypoints:
```

Comment:
Can you do this in one loop?
As you'll see later on when we study Big O notation, nested loops are not very performant.